app/controllers/issues/evidence_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Issues::EvidenceController < AuthenticatedController
```

Comment:
we discussed elsewhere that if we have an issues/evidence controller, now the rendering of the table does belong to this controller's #index as a :fetch post Issues#show is fully rendered.

That's a separate PR though, so you can make that first and the the data tables. It feels a better order than the reverse that would have you creating new code in a known-bad location to later move the code to the good location.